src/Components/VideoPost.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import '../CSS/VideoPost.css';
+
+const VideoPost = () => {
+  return (
+    <>
```

Comment:
<VideoPost /> and <VideoImage /> component are almost same combine them into one and add a wrapper component to make changes for <VideoPost />